yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -93,7 +93,12 @@ class FlowableOperator : public Flowable<D> {
     // Subscriber.
 
     void onSubscribeImpl() override {
-      subscriber_->onSubscribe(this->ref_from_this(this));
+      if (auto s = subscriber_.load()) {
+        s->onSubscribe(this->ref_from_this(this));
+      }
+      else {
+        CHECK(false) << "subscriber_ is null";
```

Comment:
Why not just crash?  The `CHECK(false)` doesn't really get you much.
```
  auto s = subscriber_.load();
  s->onSubscribe(...);
```

Also `CHECK(false)` should pretty much be `LOG(FATAL)` always.